build/depvers.sh

```diff
@@ -56,9 +58,6 @@ for (( i=0; i < ${#pkginfo[@]}; i+=2 )); do
   if test "${git}" -eq 1; then
     vers=$(git -C "${dir}" rev-parse HEAD 2>/dev/null )
   else
-    # file ownership sometimes doesn't work in docker and hg gets grumpy about
-    # this file, which is basically empty anyway, so just delete it
-    rm "${dir}/.hg/hgrc"
```

Comment:
> If the change is that we no longer have any hg-based dependencies, then why was the rm removed while the rest of hg support was kept?

I left this alone just because I didn't author that portion of this file.

> If we add another hg dependency in the future, won't we need this line again?

Yeah, we will, but we'll need a more complete solution to this problem. Basically any interaction with hg can cause problems, including the `go get -u` incantation, and this line doesn't address that. So for now, we don't really have good support for `hg` dependencies, but at least we don't pretend to deal with that problem here.